src/manifest.json

```diff
@@ -10,10 +10,10 @@
         "48": "icon-48.png",
         "128": "icon-128.png"
     },
-    "incognito": "split",
```

Comment:
What's wrong with split?

Reply:
Unfortunately without switching to the default "spanning", the extension runs completely separate service workers for incognito windows vs normal windows so any commands dispatched from an incognito window are unable to be sent to a spotify tab in another normal window.

Reply:
I just tested from within the incognito service worker devtools on the current 1.2.1 version and `await chrome.tabs.query({ url: 'https://open.spotify.com/*' })` always returns an empty array unless a spotify tab exists somewhere within an incognito window.

Reply:
Was this change ok? I think everything else is resolved, but I can't think of any other way around this one.

Reply:
I don't have a strong opinion. Having this extension work standalone in incognito is not a high priority. If you need to remove `split` for this to work it's okay with me.

Reply:
Yeah I can't imagine that many people would be trying to run separate normal/incognito Spotify tabs at the same time, and if we just want everything to work seamlessly otherwise I'm inclined to leave it out.

If there are any privacy / security concerns, they already have to explicitly go in to the extension details page and allow in incognito anyway 👍.